scripts/sct_label_utils.py

```diff
@@ -681,195 +683,243 @@ def get_parser():
+        '-i',
+        metavar=Metavar.file,
+        required=True,
+        help="Input image. Example: t2_labels.nii.gz"
+    )
+
+    optional = parser.add_argument_group("\nOPTIONAL ARGUMENTS")
+    optional.add_argument(
+        "-h",
+        "--help",
+        action="help",
+        help="Show this help message and exit."
+    )
+    optional.add_argument(
+        '-add',
+        metavar=Metavar.int,
+        type=int,
+        help="Add value to all labels. Value can be negative."
+    )
+    optional.add_argument(
+        '-create',
+        metavar=Metavar.list,
+        type=list_type(':', Coordinate),
+        help=("Create labels in a new image. List labels as: x1,y1,z1,value1:x2,y2,z2,value2."
+              "Example: 12,34,32,1:12,35,33,2")
+    )
+    optional.add_argument(
+        '-create-add',
+        metavar=Metavar.list,
+        type=list_type(':', Coordinate),
+        help=("Same as '-create', but add labels to the input image instead of creating a new image."
+              "Example: 12,34,32,1:12,35,33,2")
+    )
+    optional.add_argument(
+        '-create-seg',
+        metavar=Metavar.list,
+        type=list_type(':', str),
+        help=("R|Create labels along cord segmentation (or centerline) defined by '-i'. First value is 'z', second is "
+              "the value of the label. Separate labels with ':'. Example: 5,1:14,2:23,3. \n"
+              "To select the mid-point in the superior-inferior direction, set z to '-1'. For example if you know that "
+              "C2-C3 disc is centered in the S-I direction, then enter: -1,3")
+    )
+    optional.add_argument(
+        '-create-viewer',
+        metavar=Metavar.list,
+        type=list_type(',', int),
+        help="Manually label from a GUI a list of labels IDs, separated with ','. Example: 2,3,4,5"
+    )
+    optional.add_argument(
+        '-ilabel',
+        metavar=Metavar.file,
+        help="File that contain labels that you want to correct. It is possible to add new points with this option. "
+             "Use with -create-viewer. Example: t2_labels_auto.nii.gz"
+    )
+    optional.add_argument(
+        '-cubic-to-point',
+        action="store_true",
+        help="Compute the center-of-mass for each label value."
+    )
+    optional.add_argument(
+        '-display',
+        action="store_true",
+        help="Display all labels (i.e. non-zero values)."
+    )
+    optional.add_argument(
+        '-increment',
+        action="store_true",
+        help=("Takes all non-zero values, sort them along the inverse z direction, and attributes the values "
+              "1, 2, 3, etc.")
+    )
+    optional.add_argument(
+        '-vert-body',
+        metavar=Metavar.list,
+        type=list_type(',', int),
+        help=("R|From vertebral labeling, create points that are centered at the mid-vertebral levels. Separate "
+              "desired levels with ','. Example: 3,8\n"
+              "To get all levels, enter '0'.")
```

Comment:
```suggestion
              "To get all levels, enter 0.")
```
in case people write '0'